src/shinylive.ts

```diff
@@ -181,7 +181,7 @@ export async function shinyliveSaveAppFromUrl(): Promise<void> {
 
   const doc = await vscode.workspace.openTextDocument(localFiles[0]);
 
-  await vscode.window.showTextDocument(doc, 2, false);
+  await vscode.window.showTextDocument(doc, undefined, false);
```

Comment:
Another small quality of life change that's worth getting in: when we open the primary file of a Shinylive app after saving it locally, we now open the file in the current column (instead of the 2nd column).